airsonic-main/src/main/java/org/airsonic/player/service/MediaScannerService.java

```diff
@@ -71,7 +84,12 @@
     private ArtistDao artistDao;
     @Autowired
     private AlbumDao albumDao;
-    private int scanCount;
+    
+    @Autowired
+    @Value("#{ systemProperties['MEDIASCANNER_PARALLELISM'] ?: T(java.lang.Runtime).getRuntime().availableProcessors() + 1 }")
+    private int scannerParallelism;
```

Comment:
Parallelization config option
- Default: Processor cores + 1
- Can customize with the environment variable `MEDIASCANNER_PARALLELISM`

Reply:
Since we already have most of our settings configured via the `.properties` file, shouldn't we also use that here for consistency?

Reply:
Thought `systemProperties` took `.properties` files into account, but maybe not. I'll move this to SettingsService just to be safe as you suggested